.azure-pipelines/all.yml

```diff
@@ -0,0 +1,25 @@
+name: IntegrationTests
+trigger:
+  batch: false
+  branches:
+    include:
+    - master
+pr:
+  branches:
+    include:
+    - master
+
+jobs:
+- job: IntegrationTests
+  pool:
+    vmImage: "Ubuntu-16.04"
+  container:
+    image: golang:latest
+  steps:
+    - script: make testacc
+      displayName: Run integration tests
+      env:
+        CI: "true"
```

Comment:
Do we also need RECORD=true or does this handle that?

Reply:
Using `make testacc` actually does the right thing automatically. It runs tests against the real API without re-recording cassettes.

Reply:
Ahh ok thanks!